recipes/sktime/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "sktime" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/alan-turing-institute/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ac581f3cd3d0c17e81ac83c827ca1db7b2938d6d60deb13835c95dfd0913114a
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  run_exports:
+    - matplotlib >=3.3.2
+    # - pmdarima >=1.8.0
+    - scikit-posthocs >= 0.6.5
+    - seaborn >=0.11.0
+    - tsfresh >=0.17.0
+    # - catch22 >=0.1.0
+    - hcrystalball >=0.1.9
+    - stumpy >=1.5.1
+    # - tbats >=1.1.0
+    - fbprophet >=0.7.1
```

Comment:
This is unusual, what's the reason?

Reply:
They're optional dependencies, but if installed those are the versions that upstream claims to work with

Reply:
I think you want `run_constrained`. Not `run_exports`

Reply:
🤦 - thanks @isuruf!